### PR TITLE
dev: add note on REQUEST_MESSAGE to requesting data document

### PR DESCRIPTION
**Pull request by peterbarker**
Also a note about MAVProxy

### CODE REVIEW

File: dev/source/docs/mavlink-requesting-data.rst [hunk 1]
```diff
@@ -9,6 +9,7 @@ The ground station or companion computer can request the data it wants (and the
    - Set the ``SRx_`` parameters to cause the autopilot to pro-actively send groups of messages on start-up.  This method is easy to set-up for a small number of drones but is not recommended for most applications.
    - Send `REQUEST_DATA_STREAM <https://mavlink.io/en/messages/common.html#REQUEST_DATA_STREAM>`__ messages to set the rate for groups of messages
    - Send a `SET_MESSAGE_INTERVAL <https://mavlink.io/en/messages/common.html#MAV_CMD_SET_MESSAGE_INTERVAL>`__ command (within a `COMMAND_LONG <https://mavlink.io/en/messages/common.html#COMMAND_LONG>`__ message) to precisely control the rate of an individual message.  Note this is only support on ArduPilot 4.0 and higher
+  - Send a `_REQUEST_MESSAGE <https://mavlink.io/en/messages/common.html#MAV_CMD_GET_MESSAGE>`__ command (within a `COMMAND_LONG <https://mavlink.io/en/messages/common.html#COMMAND_LONG>`__ message) to request a single instance of a message.  Note this is only support on ArduPilot 4.0 and higher.
```
Comment (Hwurzburg) [inline dev/source/docs/mavlink-requesting-data.rst]: "supported"

Comment (peterbarker) [reply dev/source/docs/mavlink-requesting-data.rst]: Done.  And fixed where I copied it from.

Comment (Hwurzburg) [reply dev/source/docs/mavlink-requesting-data.rst]: thanks

**Comment by Hwurzburg** [comment]
merged

**Pull request merged**